package_control/ca_certs.py

```diff
@@ -141,22 +130,21 @@ def get_system_ca_bundle_path(settings):
             is_old = stats.st_mtime < time.time() - 604800
 
         if not exists or is_empty or is_old:
-            if platform == 'darwin':
-                if debug:
+            if debug:
+                if platform == 'darwin':
```

Comment:
Yeah that's another thing I wasn't sure if I should just merge, but decided not to as they still hold, even though there's an extra layer of abstraction.
@wbond What do you think?